python/felis/check.py

```diff
@@ -159,8 +159,11 @@ def check_column(self, column_obj: _Mapping, table_obj: _Mapping) -> None:
 
         # Check UCDs of columns
         ivoa_ucd = column_obj.get("ivoa:ucd")
-        if not ucd.check_ucd(ivoa_ucd, check_controlled_vocabulary=True):
-            logger.error(f"invalid ucd for {_id}: {ivoa_ucd}")
+        if ivoa_ucd:
+            try:
+                ucd.parse_ucd(ivoa_ucd, check_controlled_vocabulary=True, has_colon=";" in ivoa_ucd)
+            except ValueError as e:
+                logger.error(f"{e} in '{ivoa_ucd}' for '{_id}'")
```

Comment:
Let's say 

... in UCD '{ivoa_ucd}' ...

Especially since the error messages from `parse_ucd` don't themselves use the term.